.vscode/settings.json

```diff
@@ -5,5 +5,8 @@
   "javascript.validate.enable": false,
   "files.associations": {
     "*.mdx": "markdown"
+  },
+  "editor.codeActionsOnSave": {
+    "source.organizeImports": false
```

Comment:
I turned this on as editor default for some typescript projects. It automatically sort the imports nicely but it also add two lines at the bottom of import lines which has conflict with our own eslint settings  (only 1 line allowed ).  Thought should just be explicit about disabling default sorting behavior here.